### ISSUE TITLE
Strange encoding  for certain rows in `ccc_compiled.csv`

### ISSUE THREAD

**Issue by 18kimn** [nonviolent-action-lab/crowd-counting-consortium#6]
On row 890 of `ccc_compiled.csv`, the raw entry is 

```
2017-01-29,"Fort Lauderdale","FL","Fort Lauderdale�Hollywood International Airport",0,"demonstration",NA,NA,"General protest",NA,NA,"end the ""Muslim ban""; immigration; anti-Trump",1,"immigration; presidency; religion",NA,200,200,200,2,"0",0,"0",0,"0",0,"0",0,NA,NA,NA,NA,NA,"http://www.nbcmiami.com/news/local/-Protest-at-Miami-International-Airport-Following-Trump-Travel-Restrictions-412087023.html",NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,NA,"26.122439","-80.137317","Fort Lauderdale","Broward County","FL","12011"
```

In R, the questionable character in "Fort Lauderdale�Hollywood International Airport" is interpreted as "\x96", and after a cursory search on [StackOverflow](https://stackoverflow.com/questions/6951592/utf8-x96-does-not-map-to-unicode-at-somefile-pl-at-line-no-321-error-in) this doesn't seem to be a valid Unicode character. This also occurs for the `location_detail` column at rows 932, 1050, 1086, 1168, 1230, 1722, 1775, 1799, and 446 other rows, and possibly for other columns as well. 

In R, basic string manipulation on rows containing characters like these fails; e.g. `tolower()` throws with error `Error in tolower(.) : invalid multibyte string 1`. Some discussion about that error [can be found here](https://stackoverflow.com/questions/4993837/r-invalid-multibyte-string), although it didn't help me much.

Could I ask what encoding the CSV is in, and if there are any recommended strategies for working around them?

**Comment by alexhanna** [comment]
Thanks, y'all.

Is there any reason that the CSV output isn't explicitly encoded as UTF-8?

**Comment by nonviolent-action-lab** [comment]
Laziness? My own ignorance? (Jay here.)

Seriously, though, I just hadn't thought about this issue beyond hitting the aforementioned snag and then resolving it as described. It looks like I could do what you suggest by including `...fileEncoding = "UTF-8"` as an option in the call to `write.csv()`.

Would that be helpful?

**Comment by alexhanna** [comment]
Yeah, that'd be great. Thanks Jay! And thanks for the workaround.
